modules/pubmaticBidAdapter.js

```diff
@@ -998,6 +1030,15 @@ export const spec = {
     _handleEids(payload, validBidRequests);
     _blockedIabCategoriesValidation(payload, blockedIabCategories);
 
+    // First Party Data
+    const commonFpd = config.getConfig('ortb2') || {};
+    if (commonFpd.site) {
+      utils.mergeDeep(payload, {site: commonFpd.site});
+    }
+    if (commonFpd.user) {
+      utils.mergeDeep(payload, {user: commonFpd.user});
+    }
+
```

Comment:
why not device?

Reply:
fpd module may have several writes to device object #6099

Reply:
update: harshad slacked me to indicate a future pr might include device if it becomes popular to set and that the PBS adapter also lacks device.